### PR TITLE
WIP initial pass, equality operator 

### PR DESCRIPTION
**Pull request by DnOberon**

### CODE REVIEW

File: src/data_storage/filter.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    and() {
+        this._rawQuery.push("AND")
+        return this
+    }
+
+    query(fieldName: string, operator: string, value: any) {
+        switch(operator) {
+            case "eq": {
+               this._rawQuery.push(`${fieldName} = $${this._values.length+1}`);
```
Comment (jerenmb) [inline src/data_storage/filter.ts:32]: What does the $$ do here? Also does adding value to _values not need to happen first?

File: src/data_storage/graph/node_filter.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import {NodeT} from "../../types/graph/nodeT";
+import Result from "../../result";
+
+export default class NodeFilter extends Filter {
+    constructor() {
+        super(NodeStorage.tableName);
+    }
+
+    metatypeID(operator: string, value: any) {
+        super.query("metatype_id", operator, value)
```
Comment (jerenmb) [inline src/data_storage/graph/node_filter.ts:12]: Is operator defined?

Comment (DnOberon) [reply src/data_storage/graph/node_filter.ts:12]: Not sure what you mean. Operator will be the supported list of operators like `eq` and `neq`

File: src/data_storage/filter.ts (hunk 1 above)
Comment (DnOberon) [reply src/data_storage/filter.ts:32]: D'oh, I could add the value to values first you're right, then no need for the `+1`. The double ampersands are a mix of the notation needed for the SQL Query config for postgres `$1` and string interpolation in typescript `${}`

File: src/data_storage/graph/node_filter.ts (hunk 2 above)
Comment (jerenmb) [reply src/data_storage/graph/node_filter.ts:12]: Oh I just totally misread the metatypeID parameters. Looks good.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+name: Bug Report
```
Comment (jerenmb) [inline .github/ISSUE_TEMPLATE/feature-request.md:2]: Looks like this template needs updating to Feature Request

**Pull request merged**